bigtable-grpc-interface/pom.xml

```diff
@@ -43,35 +43,8 @@ limitations under the License.
             <artifactId>grpc-all</artifactId>
             <version>${grpc.version}</version>
         </dependency>
-        <dependency>
-            <groupId>io.netty</groupId>
-            <artifactId>netty-all</artifactId>
-            <version>${netty.version}</version>
-        </dependency>
```

Comment:
We have a few code references to netty. We should maintain a direct reference and not just depend on transitive dependencies resolving properly for us.